include/antler/system/utils.hpp

```diff
@@ -72,8 +72,24 @@ namespace antler::system {
       std::size_t n;
 
       while ((n = fread(buff.data(), 1, buff.size(), h)) > 0) {
-         fwrite(buff.data(), 1, n, stdout);
+         fwrite(buff.data(), n, 1, stdout);

```

Comment:
Why the change? In the read, we are reading `n` chars of data, in the write we are writing one block of data with length `n`?
```
size_t fread(void *ptr, size_t size, size_t nmemb, FILE *stream);
size_t fwrite(const void *ptr, size_t size, size_t nmemb, FILE *stream);
   ...'nmemb' items of data, each 'size' bytes long...
```

```suggestion
         fwrite(buff.data(), 1, n, stdout);
```

Reply:
Updated in 09a7323